modules/steam.nix

```diff
@@ -364,6 +367,15 @@ in
         # Enable VRR controls in steam
         STEAM_GAMESCOPE_VRR_SUPPORTED = "1";
 
+        # Scaling support
+        STEAM_GAMESCOPE_FANCY_SCALING_SUPPORT = "1";
+
```

Comment:
Unsure what this does... It might be adding a slider in the quick access menu to scale the interface when connected to an external display.

Reply:
- https://www.steamdeckgaming.net/post/steam-os-3-5-upcoming-changes

Under *New Scaling Options*, NIS scaling and more interestingly, *Scaling Mode* where the image can be "fit", "stretch" or "fill". The latter I assume is like `background-size: cover;` in CSS.